src/Documentation/sidebar.json

```diff
@@ -75,7 +75,8 @@
         "label": "Share Data & Model Files",
         "slug": "share-data-and-model-files"
       },
-      "shared-development-server"
+      "shared-development-server",
+      "shared-data-registry"
```

Comment:
just `data registry` should be enough I think

Reply:
Changed.